pyproject.toml

```diff
@@ -48,13 +48,14 @@ backoff = ">=1.11.1,<3.0.0"
 SPARQLWrapper = { version = ">=1.8.5,<3.0.0", optional = true }
 pyodbc = { version = "~4.0.32", optional = true }
 oracledb = { version = "~1.0.0", optional = true }
-deltalake = { version = "~0.6.4", optional = true }
+deltalake = { version = "~0.5.8", optional = true }
+typing-extensions = { version = "^4.4.0", optional = true }
```

Comment:
Tox is having some issues resolving this dependency, so I had to include it explicitly.